documentation/docs/02-runes/04-$effect.md

```diff
@@ -2,7 +2,7 @@
 title: $effect
 ---
 
-Effects are what make your application _do things_. When Svelte runs an effect function, it tracks which pieces of state (and derived state) are accessed, and re-runs the function when that state later changes.
+Effects are what make your application _do things_. When Svelte runs an effect function, it tracks which pieces of state (and derived state) are accessed (unless accessed inside [`untrack`](svelte#untrack)), and re-runs the function when that state later changes.
```

Comment:
`Effects are what make your application do things` This sounds too broad, you can actually make application that do things without effect. I suppose `react` is out of question, why not something like `$effect lets you take full control on your application`?

Reply:
> you can actually make applications do things without using effect

Like what?

Reply:
Like anything that don't use effect... like 90% of the example of the docs and tutorials? Is this a real question?

Reply:
Do you mean like the `{name}` in `<h1>hello {name}</h1>`? Because that is an effect. This is made quite clear in the following sentence

Reply:
Documenting the implementation is pointless in a user docs. This makes the sentence even more confusing. This reads:

"Effects is everywhere, you and Svelte cannot do anything without effects. **Avoid overusing effects!** 

This screams `read the full page or you will miss out` to the uninitiated. Also this is a direct contradiction with the `when not to use effect` section which heavily temper this statement.